### DISCUSSION TITLE
GDL Workbench not starting

### DISCUSSION THREAD

**Discussion by 104TMR** [gnudatalanguage/gdl#1522] [General]
Hi All,
I've downloaded the latest Windows release of GDL (gdlsetup-Windows-x86_64-standard.exe) and installed it.
It has also installed the GDL Workbench, but when I try and start Workbench, it comes up with the following Java exit error 13:
![image](https://user-images.githubusercontent.com/104307254/220795715-fe0b521e-c5e5-42e0-8e9d-3cbfa540936b.png).
What should I do to correct this and get Workbench working?
Any help appreciated.

**Comment by GillesDuvert** [comment]
Works for me (available installer as of today on GH). Windows 11 in a VM on my linux Zbook, Windows 11 is up to date. A problem with Java? In this case I'm afraid nobody from our frail developer team is able to help you.
![image](https://user-images.githubusercontent.com/33936193/222694968-d8d4107a-127b-4725-a912-e223e757f165.png)

**Comment by pjb7687** [comment]
@104TMR GDL Workbench requires Java JVM >= 11 on your machine. Update JVM to run GDL Workbench.
